app/components/SearchTableLayout.tsx

```diff
@@ -22,6 +22,17 @@ export const SearchTableLayoutComponent: React.FunctionComponent<Props> = (
     isLoading
   } = props;
 
+  const noSearchResults =
+    body.props.children.length === 0 ? (
+      <span id="no-search-results">
+        <Alert variant="secondary">
+          <p>&quot;No matching results to show.&quot;</p>
+          <p>Clicking &apos;Reset&apos; will clear your search parameters.</p>
```

Comment:
:+1: I'll remove it

Reply:
It looks like those buttons have been moved up so that they are no longer aligned with the search boxes. (Done somewhere in develop, not in this PR.) I've made a bug card & added it to the backlog https://youtrack.button.is/issue/GGIRCS-1919